### PR TITLE
Update 2022-04-12-demo-live-crypto-data-streamed-with-questdb-and-grafana.mdx

### PR DESCRIPTION
**Pull request by argshook**
This PR updates the query used for `Implied BTC-ETH exchange rate`, in order to improve the performance

### CODE REVIEW

File: blog/2022-04-12-demo-live-crypto-data-streamed-with-questdb-and-grafana.mdx [hunk 1]
```diff
@@ -149,27 +149,31 @@ price of ETH-USD to get the implied BTC/ETH rate.
...
+    WHERE symbol = 'ETH-USD' AND timestamp > dateadd('m', -30, now())
+  ) timestamp(y)
+)
+SELECT
+  x AS time,
+  btcusd / ethusd AS crossPrice,
+  btcusd,
+  ethusd
+FROM btc
+ASOF JOIN eth;
```
Comment (argshook) [inline blog/2022-04-12-demo-live-crypto-data-streamed-with-questdb-and-grafana.mdx:172]:
```suggestion
    WITH btc AS (
      (
        SELECT timestamp AS x, price AS btcusd
        FROM trades
        WHERE symbol = 'BTC-USD' AND timestamp > dateadd('m', -30, now())
      ) timestamp(x)
    ),
    eth AS (
      (
        SELECT timestamp AS y, price AS ethusd
        FROM trades
        WHERE symbol = 'ETH-USD' AND timestamp > dateadd('m', -30, now())
      ) timestamp(y)
    )
    SELECT
      x AS time,
      btcusd / ethusd AS crossPrice,
      btcusd,
      ethusd
    FROM btc
    ASOF JOIN eth;
```

File: blog/2022-04-12-demo-live-crypto-data-streamed-with-questdb-and-grafana.mdx [hunk 2]
```diff
@@ -150,26 +150,30 @@ price of ETH-USD to get the implied BTC/ETH rate.
 
   ```questdb-sql title="Implied BTC-ETH exchange rate"
```
Comment (bziobrowski) [inline blog/2022-04-12-demo-live-crypto-data-streamed-with-questdb-and-grafana.mdx:151]: Using timestamp(x/y) is not necessary, so it'd be better to simplify query to : 


WITH btc AS (
    SELECT timestamp AS x, price AS btcusd
    FROM trades
    WHERE symbol = 'BTC-USD' AND timestamp > dateadd('m', -30, now())
),
eth AS (
    SELECT timestamp AS y, price AS ethusd
    FROM trades
    WHERE symbol = 'ETH-USD' AND timestamp > dateadd('m', -30, now())
)
SELECT
  x AS time,
  btcusd / ethusd AS crossPrice,
  btcusd,
  ethusd
FROM btc
ASOF JOIN eth;

Comment (argshook) [reply blog/2022-04-12-demo-live-crypto-data-streamed-with-questdb-and-grafana.mdx:151]: Thanks!

I updated the query to the one you're suggesting. Please check

**Review by bziobrowski — APPROVED** [review]
👍

**Pull request merged**